tasks/83_missing-term/missing-term.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
+// I don't know if the way I calculated step in diffs array is correct,
```

Comment:
Generally you also can solve it by loop from 1 element and if `arr[i] - arr[i - 1] !== arr[i + 1] - arr[i]` than missed element will equal `arr[i] + smallestDiff`